migration/src/pages.rs

```diff
@@ -0,0 +1,50 @@
+use csv::ReaderBuilder;
+use dailp::page::ContentBlock;
+use dailp::page::Markdown;
+use dailp::page::NewPageInput;
+use dailp::{page::Page, Database};
+use serde::Deserialize;
+use std::fs::File;
+
+// CSV row structure (includes all fields even though we skip some)
+#[derive(Debug, Deserialize)]
+struct CsvRow {
+    #[serde(rename = "page_id")]
+    _page_id: String,
+    #[serde(rename = "slug")]
+    _slug: String,
+    path: String,
+    title: String,
+    content: String,
+}
+
+pub fn load_pages(file_path: &str) -> Result<Vec<NewPageInput>, anyhow::Error> {
+    let file = File::open(file_path)?;
+    let mut reader = ReaderBuilder::new().from_reader(file);
+
+    let mut pages = Vec::new();
+    for result in reader.deserialize() {
```

Comment:
You should add some explicit error handling for CSV parse issues. What happens if a column is missing? if it is malformed?